exporters/variables-scss/src/helpers/tokenHelper.ts

```diff
@@ -94,8 +132,23 @@ export const typographyValue = (
     `font-weight: ${fontWeight.text}`,
   ];
 
+  const baseJsStyles = [
```

Comment:
Just an idea, but what about defining this object once and then applying `toCamelCase` on the keys of the object in some kind of map function? So will need to repeat the same object twice :-)

Reply:
Ok, tomorrow. :-)